package.json

```diff
@@ -65,9 +65,16 @@
     "stream-browserify": "^2.0.1",
     "stylelint": "^8.0.0",
     "stylelint-config-standard": "^17.0.0",
+    "upath": "^1.2.0",
```

Comment:
What's the reason?

Reply:
A test was failing because the path was not correct. This normalize the path on Windows.

Reply:
I'm too lazy to set up git on windows to check. Especially, since there are WSL these days available for windows developers to have a normal Linux environment. 

Based on your changes, this is the only place that windows backslash might complain about.
![image](https://user-images.githubusercontent.com/1775318/80524884-37321800-8945-11ea-9b54-9256f5279671.png)

But since this module is literally deprecated, I will just merge this in. :-)